modules/index-service/src/main/java/org/opencastproject/index/service/catalog/adapter/ConfigurableDCCatalogUIAdapter.java

```diff
@@ -0,0 +1,181 @@
+ * the License.
+ *
+ */
+package org.opencastproject.index.service.catalog.adapter;
+
+import static org.opencastproject.index.service.catalog.adapter.CatalogUIAdapterFactory.CONF_FLAVOR_KEY;
+import static org.opencastproject.index.service.catalog.adapter.CatalogUIAdapterFactory.CONF_ORGANIZATION_KEY;
+import static org.opencastproject.index.service.catalog.adapter.CatalogUIAdapterFactory.CONF_TITLE_KEY;
+import static org.opencastproject.util.OsgiUtil.getCfg;
+
+import org.opencastproject.index.service.resources.list.api.ListProvidersService;
+import org.opencastproject.mediapackage.EName;
+import org.opencastproject.mediapackage.MediaPackageElementFlavor;
+import org.opencastproject.metadata.dublincore.CatalogUIAdapter;
+import org.opencastproject.metadata.dublincore.DublinCore;
+import org.opencastproject.metadata.dublincore.DublinCoreCatalog;
+import org.opencastproject.metadata.dublincore.DublinCoreValue;
+import org.opencastproject.metadata.dublincore.MetadataCollection;
+import org.opencastproject.metadata.dublincore.MetadataField;
+import org.opencastproject.metadata.dublincore.SeriesCatalogUIAdapter;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.osgi.service.cm.ConfigurationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Dictionary;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public abstract class ConfigurableDCCatalogUIAdapter implements CatalogUIAdapter {
+
+  private static final Logger logger = LoggerFactory.getLogger(ConfigurableDCCatalogUIAdapter.class);
+
+  /** The catalog UI adapter configuration */
+  protected CatalogUIAdapterConfiguration config;
+
+  /** The organization name */
+  protected String organization;
+
+  /** The flavor of this catalog */
+  protected MediaPackageElementFlavor flavor;
+
+  /** The title of this catalog */
+  protected String title;
+
+  /** The metadata fields for all properties of the underlying DublinCore */
+  protected Map<String, MetadataField<?>> dublinCoreProperties;
+
+  /** Reference to the list providers service */
+  protected ListProvidersService listProvidersService;
+
+  /**
+   * Reconfigures the {@link SeriesCatalogUIAdapter} instance with an updated set of configuration properties;
+   *
+   * @param properties
+   *          the configuration properties
+   * @throws ConfigurationException
+   *           if there is a configuration error
+   */
+  public void updated(Dictionary<String, ?> properties) throws ConfigurationException {
+    config = CatalogUIAdapterConfiguration.loadFromDictionary(properties);
+    organization = getCfg(properties, CONF_ORGANIZATION_KEY);
+    flavor = MediaPackageElementFlavor.parseFlavor(getCfg(properties, CONF_FLAVOR_KEY));
+    title = getCfg(properties, CONF_TITLE_KEY);
+    dublinCoreProperties = DublinCoreMetadataUtil.getDublinCoreProperties(properties);
```

Comment:
From an operational perspective, this should be logged as INFO (as it did before) as it changes the state of the application which is relevant if you want to understand what Opencast does based on its log files.